content/docs/reference-events.md

```diff
@@ -31,44 +31,44 @@ number timeStamp
 string type
 ```
 
-> Note:
+> Catatan:
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> Sejak v0.14, mengembalikan nilai `false` dari _event handler_ tidak akan menghentikan propagasi _event_. Sebagai gantinya, `e.stopPropagation()` atau `e.preventDefault()` perlu dipanggil secara manual.
 
 ### Event Pooling {#event-pooling}
 
-The `SyntheticEvent` is pooled. This means that the `SyntheticEvent` object will be reused and all properties will be nullified after the event callback has been invoked.
-This is for performance reasons.
-As such, you cannot access the event in an asynchronous way.
+`SyntheticEvent` dikumpulkan. Hal ini berarti objek `SyntheticEvent` akan digunakan kembali dan isi objek akan dikosongkan setelah _callback_ untuk _event_ tersebut dipanggil.
+Hal ini dilakukan untuk mengoptimalkan kinerja.
+Dengan demikian, _event_ tersebut tidak dapat diakses secara asinkron.
 
 ```javascript
 function onClick(event) {
-  console.log(event); // => nullified object.
-  console.log(event.type); // => "click"
-  const eventType = event.type; // => "click"
+  console.log(event); // => objek yang dikosongkan.
+  console.log(event.type); // => "klik"
+  const eventType = event.type; // => "klik"
 
   setTimeout(function() {
     console.log(event.type); // => null
-    console.log(eventType); // => "click"
+    console.log(eventType); // => "klik"
   }, 0);
 
-  // Won't work. this.state.clickEvent will only contain null values.
+  //Tidak akan berfungsi. this.state.clickEvent hanya berisi nilai kosong.
   this.setState({clickEvent: event});
 
-  // You can still export event properties.
+  // Anda masih dapat mengekspor atribut event.
   this.setState({eventType: event.type});
 }
 ```
 
-> Note:
+> Catatan:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> Jika Anda ingin mengakses isi _event_ secara asinkron, Anda perlu memanggil fungsi `event.persist()` dalam _event_, yang akan mengeluarkan _synthetic event_ dari kumpulan sehingga acuan terhadap _event_ tersebut dapat dipertahakan melalui kode pengguna.
```

Comment:
```suggestion
> Jika Anda ingin mengakses isi _event_ secara asinkron, Anda perlu memanggil fungsi `event.persist()` dalam _event_, yang akan mengeluarkan _synthetic event_ dari kumpulan (*pool*) sehingga acuan terhadap _event_ tersebut dapat dipertahakan melalui kode pengguna.
```

I think because in the previous section we don't translate `Event pooling` we need to give `kumpulan` some context by adding `(*pool*)`

also

dipertahakan -> dipertahankan looks like a typo 😄

Reply:
I agree with this, however do you think that "pool" still needs to be translated to "kumpulan" in this case?

Also fixed the typo :smile:

Reply:
IMO It's ok to be translated 😄

Reply:
:+1: